st2common/st2common/models/db/liveaction.py

```diff
@@ -59,6 +59,9 @@ class LiveActionDB(stormbase.StormFoundationDB):
     action = me.StringField(
         required=True,
         help_text='Reference to the action that has to be executed.')
+    is_action_workflow = me.BooleanField(
```

Comment:
Yeah, I was thinking about it, but the attribute refers to the related action and not LiveAction. Another option also is `action_is_workflow` :P

WDYT?

We could also do something like we do with `runner_info` attribute (e.g. `action_info`), but I'm not too big of a fan of free for all dictionaries since it's hard to see what are possible attributes, etc.